src/evo/assetlocktx.cpp

```diff
@@ -115,14 +122,9 @@ bool CAssetUnlockPayload::VerifySig(const uint256& msgHash, const CBlockIndex* p
 
     Consensus::LLMQType llmqType = Params().GetConsensus().llmqTypeAssetLocks;
 
-    const auto& llmq_params_opt = llmq::GetLLMQParams(llmqType);
-    if (!llmq_params_opt) {
-        return state.Invalid(TxValidationResult::TX_CONSENSUS, "bad-assetunlock-llmq-type");
-    }
-
-    // We check at most 2 quorums, so, count is equal to 2
-    const int count = 2;
-    auto quorums = llmq::quorumManager->ScanQuorums(llmqType, pindexTip, count > -1 ? count : llmq_params_opt->signingActiveQuorumCount);
+    // We check at most 2 quorums, so, quorum_count is equal to 2
+    const int quorum_count = 2;
+    auto quorums = llmq::quorumManager->ScanQuorums(llmqType, pindexTip, quorum_count);
```

Comment:
`quorum_count` isn't used anywhere below besides `ScanQuorums`. Why even declaring it? If you want to be clear about the meaning of `2` you can do smth like that instead
```suggestion
    // We check at most 2 quorums
    auto quorums = llmq::quorumManager->ScanQuorums(llmqType, pindexTip, 2 /* nCountRequested */);
```

Reply:
I just don't want really hanging literal that even doesn't have a type, but it doesn't really matter/useful.